optimize/client/src/modules/filter/FilterList.js

```diff
@@ -387,20 +379,12 @@ FilterList.defaultProps = {
   deleteFilter: () => {},
 };
 
-function isDecisionVariable(type) {
-  return ['inputVariable', 'outputVariable'].includes(type);
-}
-
 function getVariableLabel(nameOrId, type, filterType, variables) {
```

Comment:
I think we dont need this function, we can keep just the `getProcessVariableLabel`